assets/sass/components/adminbar/_googlesitekit-adminbar.scss

```diff
@@ -66,8 +88,7 @@
 	}
 
 	// Display sub menu on hover.
-	.googlesitekit-wp-adminbar.hover .googlesitekit-adminbar,
-	&.nojs .googlesitekit-wp-adminbar:hover .googlesitekit-adminbar {
+	&:not(.nojs) .googlesitekit-wp-adminbar:hover .googlesitekit-adminbar {
```

Comment:
I'm not sure that this should be changed. The noscript html will be displayed in the submenu on hover so we shouldn't need to prevent this from displaying.

Reply:
adminbar doesn't show by default. It would only appear on hover. So I've just added only show if parent doesn't have `.nojs` and hover.